### ISSUE TITLE
Make the Ban response, undo button its own composable 

### ISSUE THREAD

**Issue by thePlebDev** [thePlebDev/Clicker#536]
# Proposed change
- I really don't like how the undo ban button is inside of the ScrollToBottom composable 
- I want to remove it, place it in its own composable and allow the user to move it where ever they want

# Why is this important
- This will offer the user a better overall experience 

# Additional context
- n/a

**Comment by thePlebDev** [comment]
Here is the old code that I took out of the ScrollToBottom composable:
```
        when (val response = banResponse) {
            is Response.Loading -> {}
            is Response.Success -> {
                if (response.data && undoBanResponse) {
                }
                if (response.data && !undoBanResponse) {
                    Icon(
                        imageVector = Icons.Default.Refresh,
                        contentDescription = stringResource(R.string.undo_ban_button),
                        modifier = Modifier
                            .clip(RoundedCornerShape(5.dp))
                            .size(50.dp)
                            .clickable { undoBan() } // todo: WHEN CLICKED WE NEED TO CLOSE ALL THE MODAL
                            .align(Alignment.BottomEnd)
                            .background(Color.White),
                        tint = Color.Magenta
                    )
                    LaunchedEffect(key1 = Unit) {
                        delay(10000)
                        // todo: reset the response.data && !undoBanResponse
                        removeUndoButton()
                    }
                }
            }
            is Response.Failure -> {}
        }
```
- Very ugly but we can work on it later. Right now I have to go back to reworking the chat composable

**Comment by thePlebDev** [comment]
- closed with #576
